client/src/components/text.zig

```diff
@@ -42,53 +97,21 @@ fn at_impl(
     var mouseOnText = false;
```

Comment:
This doesn't need to be a `var`. We never mutate, other than in the `if` that immediately follows this line.

Reply:
I don't think that is true. Check line 107:
```zig
    drawTextBox(textBoxPosition, textBoxSize, if (mouseOnText) config.ColorPalette.secondary else rl.Color.white)
```

Reply:
Which is not a mutation of `mouseOnText`.